src/status_im2/contexts/chat/lightbox/zoomable_image/utils.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im2.contexts.chat.lightbox.zoomable-image.utils
+  (:require
+    [clojure.string :as string]
+    [react-native.orientation :as orientation]
+    [react-native.platform :as platform]
+    [status-im2.contexts.chat.lightbox.zoomable-image.constants :as c]
+    [utils.re-frame :as rf]))
+
+;;; Calculates all required dimensions
+;;; Dimensions calculations are different on iOS and Android because landscape mode is implemented
+;;; differently.
+;;; On Android, we just need to resize the content, and the OS takes care of the animations.
+;;; On iOS, we need to animate the content ourselves in code
+(defn get-dimensions
```

Comment:
@OmarBasem, please move these comments on top of functions to their docstrings. The same feedback applies to `handle-exit-lightbox-signal` and `handle-zoom-out-signal`.